api/api.go

```diff
@@ -26,14 +26,21 @@ var (
 	// metric api.iters_to_points is how long it takes to decode points from a chunk iterator
 	itersToPointsDuration = stats.NewLatencyHistogram15s32("api.iters_to_points")
 
-	// metric api.request_handle is how long it takes to handle a render request
-	reqHandleDuration = stats.NewLatencyHistogram15s32("api.request_handle")
-
 	// metric api.requests_span.mem_and_cassandra is the timerange of requests hitting both in-memory and cassandra
 	reqSpanBoth = stats.NewMeter32("api.requests_span.mem_and_cassandra", false)
 
 	// metric api.requests_span.mem is the timerange of requests hitting only the ringbuffer
 	reqSpanMem = stats.NewMeter32("api.requests_span.mem", false)
+
+	// metric api.render.series is the number of series a /render request is handling.  This is the number
+	// of metrics after all of the targets in the request have expaned by searching the index.
+	reqRenderSeriesCount = stats.NewMeter32("api.render.series", false)
+
+	// metric api.render.series is the number of targets a /render request is handling.
+	reqRenderTargetCount = stats.NewMeter32("api.render.targets", false)
```

Comment:
this is the number of target arguments (possibly patterns) coming in directly from the request, right? might want to clarify that.